utils/coreutils/techutils_test.go

```diff
@@ -364,14 +374,46 @@ func TestCleanSubDirectories(t *testing.T) {
 	}
 }
 
+func createTempDirWithPyProjectToml(t *testing.T, tech Technology) (tmpDir string, callback func()) {
```

Comment:
can we add a test for a project with multiple descriptors (specifically with PyProjectToml) but that one of them isn't python? e.g - requirements.txt, package.json and PyProject.toml - could be interesting to validate...